cves/kernel/CVE-2019-19069.yml

```diff
@@ -106,7 +104,8 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 6cffd79504ce040f460831030d3069fa1c99bb71
-  note: Discovered automatically by archeogit.
+  note: |
+   This commit involves the import of the structure that is used as a parameter for the method involved in the vulnerability.
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
```

Comment:
I would give this vulnerability a five out of ten since it is just improper memory release that attackers can use to cause a denial of service